src/javascript/binary/common_functions/form_validation.js

```diff
@@ -163,7 +165,7 @@ const Validation = (() => {
         email        : { func: validEmail,        message: 'Invalid email address' },
         password     : { func: validPassword,     message: 'Password should have lower and uppercase letters with numbers.' },
         general      : { func: validGeneral,      message: 'Only letters, numbers, space, hyphen, period, and apostrophe are allowed.' },
-        address      : { func: validAddress,      message: 'Only letters, numbers, space, hyphen, period, and apostrophe are allowed.' },
+        address      : { func: validAddress,      message: 'Invalid character used.' },
```

Comment:
we need to add `Invalid character used.` to translation file

Reply:
Hi negarn, I'm not completely familiar with how the localization works for this project, do I have to add the lines below:
`msgid "Invalid character used."`
`msgstr ""`
to the en.po and all the .po files in the locales folder?

Reply:
@zakibinary There is no need to update `.po` files manually.
The strings which used in javascript, need to be also included in `scripts/bin/GenerateStaticData.pm` to get involved in the translation process. So, adding new strings there is enough.

Reply:
@binary-hamedanchi , noted with thanks!